.github/workflows/release.yml

```diff
@@ -0,0 +1,203 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        description: 'Release type'     
+        required: true
+        default: 'auto' 
+        type: choice
+        options:
+          - minor
+          - patch 
+      repository:
+        description: Repository (in owner/repo format)
+        required: true
+        default: kiali/helm-charts
+        type: string
```

Comment:
Like I said in the other PR, I think this input should be removed. It should always be the repo where the workflow was triggered.